content/en/docs/reference/Design Docs/_index.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Design Docs
```

Comment:
Since the title is already clear about the content, please avoid using spaces in directory names. In other words, please try place this under a directory named "design".